docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,42 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+The Dutch government demands that government organisations are transparent about the use of
+high impact algorithms. This is a requirement of
+the [Open Government Act](https://www.rijksoverheid.nl/onderwerpen/wet-open-overheid-woo).
+This transparency can start with registering an algorithm in the
+[Algoritmeregister](https://algoritmes.overheid.nl/).
+
+Transparency and validation can be achieved by doing assessments,
+like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)
+for decision making and technical evaluations to give insight in the features and outcomes of an algorithm. These
+outcomes can be combined to create a report that can be shared.
+
+There are several tools available on the market. Some support only assessments, others already
+combine more features and can generate a report. There is however no tool that supports
+all [requirements](https://minbzk.github.io/ai-validation/Projects/TAD/Tools/) we have.
+
+## Assumptions
+
+* Collaborating or extending another project will not give us the tool we are looking for.
+* We can reuse certain components, like the
+[plugins from AIVerify](https://github.com/IMDA-BTG/aiverify/tree/main/stock-plugins) for technical tests.
+* The tool will use a loose modules design, by scanning directories, to gather plugin functionality.
```

Comment:
I think this sentence needs to be split up into multiple sentences or item, because I am not sure how I should read this now. 

I also would use another word than "loose modules design". Maybe something like "loosely coupled module design" or "The tool will use a design based on loose modules."

Reply:
I reworded the sentence.